pages/hud.coffee

```diff
@@ -86,7 +86,7 @@ handlers =
     countElement.id = "hud-match-count"
     countElement.style.float = "right"
     hud.appendChild countElement
-    Utils.setTimeout 0, -> inputElement.focus()
+    Utils.setTimeout 17, -> inputElement.focus()
```

Comment:
Why 17ms -- because that's the duration of 1 frame at 60 fps? Let's move this value into a variable constant (e.g. FRAME_DURATION) with a description explaining why it's 17ms.

Reply:
Firstly, thanks.

During my tests on Chrome 74.0.3729.108 x64, Win10, with blank user data folder, a 0~1ms timeout was not enough. Although the success rate then seemed >= 90%, I could still reproduce the fails every a few retries.

It is indeed because 17ms is a frame interval that I choose it. The ideas behind it are: 1. the time span should be long enough for operating system's message receiving (IPC) race conditions; 2. not so long that users may feel it. Therefore, I think any time span among 10ms~50ms may be OKay. And during the further tens of retries, 17ms worked well.

BTW, maybe the failing of 1ms is because my PC was doing other heavy work like video playing / code compiling - an operating system may be busy in other tasks, and it does need some time to wake up a process / thread, so 1ms is not so safe for inter-process communications.

I'll extract it to a constant.